maestro-cli/src/main/java/maestro/cli/runner/TestSuiteInteractor.kt

```diff
@@ -0,0 +1,143 @@
+package maestro.cli.runner
+
+import maestro.Maestro
+import maestro.cli.device.Device
+import maestro.cli.model.FlowStatus
+import maestro.cli.util.PrintUtils
+import maestro.cli.view.TestSuiteStatusView
+import maestro.cli.view.TestSuiteStatusView.TestSuiteViewModel
+import maestro.orchestra.Orchestra
+import maestro.orchestra.yaml.YamlCommandReader
+import java.io.File
+
+object TestSuiteInteractor {
+
+    fun runTestSuite(
+        maestro: Maestro,
+        device: Device?,
+        input: File,
+        env: Map<String, String>,
+    ): SuiteResult {
+        return if (input.isFile) {
+            runTestSuite(
+                maestro, device, listOf(input), env
+            )
+        } else {
+            val flowFiles = input
+                .listFiles()
+                .filter {
+                    it.isFile
+                        && it.extension in setOf("yaml", "yml")
+                        && it.nameWithoutExtension != "config"
```

Comment:
Let's also ignore files starting with underscore. That way users can quickly disable tests.

Reply:
I am planning to introduce an `ignore: true` in flows later on so that we can have an explicit support for that